pkg/typesystem/typesystem.go

```diff
@@ -389,6 +408,35 @@ func (t *TypeSystem) IsDirectlyRelated(target *openfgav1.RelationReference, sour
 	return false, nil
 }
 
+// TTUResolvesExclusivelyToDirectlyAssignable returns whether the computedRelation of object's tupleRelation is directly assignable.
+func (t *TypeSystem) TTUResolvesExclusivelyToDirectlyAssignable(objectType, tuplesetRelation, computedRelation string) (bool, error) {
+	tuplesetRelationTypes, directlyAssignable, err := t.resolvesTypeRelationToDirectlyAssignable(objectType, tuplesetRelation)
+	if err != nil {
+		return false, err
+	}
+	if !directlyAssignable {
+		return false, nil
+	}
+	var relationUndefinedError *RelationUndefinedError
```

Comment:
Seems this could simply be a const

Reply:
Checked - this cannot be constant since errors.As will not work.